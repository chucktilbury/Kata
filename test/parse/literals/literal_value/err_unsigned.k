;-------------------------------------------------------------------------------
; Test the scanner's ability to detect an invalid number in 0x123abc format.
;-------------------------------------------------------------------------------

var unsigned some_name = 0xx123
