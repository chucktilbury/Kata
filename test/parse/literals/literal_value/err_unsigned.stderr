Syntax: /home/chuck/Src/Kata/test/parse/literals/literal_value/err_unsigned.k: 5: 28: expected an assignment item but got : malformed hex number, expected hex digit, space or operator
